docs/automation.md

```diff
@@ -0,0 +1,133 @@
+# Automation
+
+![beta](assets/beta.svg)
+
+> v2.8 and after
+
+This is guide on automation. 
+
+## Environment Variables
+
+To do any automation you need to get the following environment variables:
+
+* `ARGO_SERVER` - the hostname and port of your server, e.g. `argo-server:2746`
+* `ARGO_TOKEN` - an [access token](access-token.md).
+
+See `argo --help` to learn more.
+
+## Waiting For External Events
```

Comment:
This is confusing. 

I thought it was about event triggering, but it is actually `resume a workflow through CLI and API`